### ISSUE TITLE
Bug: Assembly inclusions is not fully working

### ISSUE THREAD

**Issue by Chuckame** [wvengen/proguard-maven-plugin#67]
When we have multiple artifactId with the same artifactId, only the first found artifactId match.
This example will be collect only `lib-one` dependency :
```xml
<dependency>
   <groupId>my.group.id</groupId>
   <artifactId>lib-one</artifactId>
</dependency>
<dependency>
   <groupId>my.group.id</groupId>
   <artifactId>lib-two</artifactId>
</dependency>
...
<plugin>
  <artifactId>proguard-maven-plugin</artifactId>
  <assembly>
    <inclusions>
      <groupId>my.group.id</groupId>
      <artifactId>*</artifactId>
    </inclusions>
  </assembly>
</plugin>
```

Because of ProGuardMojo#getDependency : https://github.com/wvengen/proguard-maven-plugin/blob/f0ec275312f039978fa0f46f3d2b497f66ad4101/src/main/java/com/github/wvengen/maven/proguard/ProGuardMojo.java#L849
This method only returns the first matched artifact.

**Comment by cjgriscom** [comment]
@lasselindqvist I believe this issue is resolved now by #68

**Comment by lasselindqvist** [comment]
Thanks. The new getDependencies method iterates over all dependencies and adds all of them.
